src/cook-web/src/app/admin/operations/page.tsx

```
@@ -1,52 +1,1282 @@
+        }
+        setPageIndex(targetPage);
+        if (err instanceof ErrorWithCode) {
+          setError({ message: err.message, code: err.code });
+        } else if (err instanceof Error) {
+          setError({ message: err.message });
+        } else {
+          setError({ message: t('common.core.unknownError') });
+        }
+      } finally {
+        if (requestId !== requestIdRef.current) {
+          return;
```

Biome (2.4.10)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)